js/src/setup-mc/setup-stepper/setup-paid-ads/product-feed-status-section.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * External dependencies
+ */
+import { sprintf, __, _n } from '@wordpress/i18n';
+import { Flex, FlexItem, FlexBlock } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import Section from '.~/wcdl/section';
+import AppDocumentationLink from '.~/components/app-documentation-link';
+import SyncIcon from '.~/components/sync-icon';
+import AppTooltip from '.~/components/app-tooltip';
+import getNumberOfSyncProducts from '.~/utils/getNumberOfSyncProducts';
+import './product-feed-status-section.scss';
+
+function ProductQuantity( { quantity } ) {
+	if ( ! Number.isInteger( quantity ) ) {
+		return null;
+	}
+
+	const text = sprintf(
+		// translators: %d: number of products will be synced to Google Merchant Center.
+		_n( '%d product', '%d products', quantity, 'google-listings-and-ads' ),
+		quantity
+	);
+
+	return (
+		<>
+			<span className="gla-product-feed-status-section__product-quantity-separator" />
+			<AppTooltip
+				className
+				position="top center"
+				text={ __(
+					'You can manage and edit your product feed after this setup',
+					'google-listings-and-ads'
+				) }
+			>
+				{ text }
+			</AppTooltip>
+		</>
+	);
+}
+
+// TODO: `href`` is not yet ready. Will be added later.
+/**
+ * @fires gla_documentation_link_click with `{ context: 'setup-paid-ads', link_id: 'product-feed-status-learn-more', href: 'https://example.com' }`
+ */
+
+/**
+ * Renders a section layout to elaborate on how the product listings will be processed
+ * and show the number of products will be synced to Google Merchant Center.
+ */
+export default function ProductFeedStatusSection() {
+	/*
+	const { data, hasFinishedResolution } = useAppSelectDispatch(
+		'getMCProductStatistics'
+	);
+	*/
+	// TODO: Replace the dummy data with the above code later to use the adjusted API.
```

Comment:
❔ Just wanted to make sure my understanding is right. The API `mc/product-statistics` would only be returning data after we call `mc/settings/sync`, so the comment `Replace the dummy data with the above code later to use the adjusted API.` means we would make a POST request to  `mc/settings/sync` later, right?

Reply:
Yes, as per my test from the frontend side, it has to call `mc/settings/sync` before getting statistics from `mc/product-statistics`, so the frontend side will call `mc/settings/sync` when entering step 4. And the `mc/product-statistics` seems to check whether the `MC_SETUP_COMPLETED_AT` flag is set before returning, thus, we will need adjustment in this project as well.